src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_RENT = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const basePrise = days * COST_RENT;
```

Comment:
There is a typo in the variable name 'basePrise'. It should be 'basePrice' to correctly represent its purpose, which is to hold the base price of the rental cost.